packages/handlebars-templates/src/index.ts

```diff
@@ -290,6 +290,30 @@ export function registerStandardHelpers<O>(hbs: typeof Handlebars, { utils }: Co
 		}
 	})
 	
+	hbs.registerHelper('ifdef', function(this: object, a: unknown, options: Handlebars.HelperOptions) {
```

Comment:
The parameter, `a`, made sense in `ifeq` as there's an `a` and `b`, but how about here we rename `a` to `value`?